src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,137 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
  * Date Modified: 10/03/17
  */
 
-public class Meal implements DietFacts {
-    @Override
-    public int getCaloriesIn() {
-        return 0;
-    }
+public class Meal implements DietFacts
+    {
+        @Override
+        public int getCaloriesIn() {
+            return 0;
+        }
 
-    @Override
-    public double getTotalFatIn() {
-        return 0;
-    }
+        @Override
+        public double getTotalFatIn() {
+            return 0;
```

Comment:
All overridden methods must have actual implementations. 
E.g. I had my lunch, and I come to you (since you have Meal object and knows information about it), and I ask - how much fat was in my lunch? 
This is what `getTotalfatIn()` should return.
Currently, no matter what I have eaten, your method return 0. So no matter of the food, my meal contains 0 fat.